dist/types/src/messages/records.d.ts

```diff
@@ -3,6 +3,7 @@ export type RecordsFilter = {
     programIds?: string[];
     functionId?: string;
     type: 'all' | 'spent' | 'unspent';
+    name?: string;
```

Comment:
Are we making this an array, per Jon's comment on the other PR? I also think it works better.